src/server/request-handlers/dashboards.js

```diff
@@ -4,51 +4,72 @@ var db = require('../db');
 
 module.exports = {
   handlePost: function (req, res, next) {
-    var users_id = req.body.users_id;
-    var repo_link = req.body.repo_link;
-    var findDashboard = "SELECT * FROM dashboards WHERE repo_link='" + repo_link.toString() + "';";
-    // Check if dashboard exists
-    db.query(findDashboard, function (err, results) {
+    var githubId = req.cookies.githubId;
+
+    var dashboardDetails = {};
+    dashboardDetails.repo_link = req.body.html_url;
+    dashboardDetails.branch = 'master';
```

Comment:
Instead of hard-coding 'Master', use `req.body.default_branch`. This will grab the actual default branch out of the git object